src/Pages/tae/Main/Components/Feeds/Article.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import ImageBT from "./ImageBT";
+
+class Article extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      newComment: "",
+      commentList: [],
+      name: "recurian1058",
+      reactionBT: [
+        {
+          BTName: "reactBT",
+          ImgName: "reactBTImg",
+          alt: "Like button",
+          src: "images/tae/reactionheart.png",
+        },
+        {
+          BTName: "reactBT",
+          ImgName: "reactBTImg",
+          alt: "Go this post",
+          src: "images/tae/reactiontalkbox.png",
+        },
+        {
+          BTName: "reactBT",
+          ImgName: "reactBTImg",
+          alt: "DM button",
+          src: "images/tae/reactionpaperplane.png",
+        },
+        {
+          BTName: "reactBTBM",
+          ImgName: "bookmark",
+          alt: "bookmark buuton",
+          src: "images/tae/bookmark.png",
+        },
+      ],
+    };
+  }
+
+  inputText = (e) => this.setState({ newComment: e.target.value });
+
+  addList = () => {
+    const { commentList, newComment } = this.state;
+    let commentListArr = commentList;
+    commentListArr.push({ text: newComment });
+    this.setState({
+      newComment: "",
+      commentList: commentListArr,
+    });
+  };
```

Comment:
```suggestion
  addCommentList = () => {
    const { commentList, newComment } = this.state;
    this.setState({
      newComment: "",
      commentList: [...commentList, {text: newComment}]
    });
  };
```

- spread 연산자 사용해서 위와 같이 바꿀 수 있겠네요!

Reply:
오 더 깔끔해졌네요